upgrade_analysis/models/upgrade_record.py

```diff
@@ -140,7 +140,7 @@ def list_modules(self):
             """SELECT DISTINCT(module) FROM upgrade_record
             ORDER BY module"""
         )
-        return [module for module, in self.env.cr.fetchall()]
+        return [module[0] for module in self.env.cr.fetchall()]
```

Comment:
Why did you change this? I think the original is a very elegant way to parse tuples of size one.

Reply:
No, without that `[0]` then you return something like [("module1",), ("module2",), ...] which later breaks. My changes have been tested.

Reply:
The comma after `module` unpacks the tuple fine.

Reply:
Oh, shit, then It was my bad. Yeah, it's elegant, but where it falls the readability? hehe